it/content/src/main/content/jcr_root/content/forms/af/core-components-it/samples/textinput/basic/.content.xml

```diff
@@ -1,51 +1,63 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
-    jcr:primaryType="cq:Page">
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0"
+          jcr:primaryType="cq:Page">
     <jcr:content
-        cq:deviceGroups="[mobile/groups/responsive]"
-        cq:lastModified="{Date}2021-12-02T15:12:59.900+05:30"
-        cq:lastModifiedBy="admin"
-        cq:template="/conf/core-components-it/settings/wcm/templates/af-blank-v2"
-        jcr:language="en"
-        jcr:primaryType="cq:PageContent"
-        jcr:title="Adaptive Form V2 (IT)"
-        sling:resourceType="forms-components-examples/components/page">
+            cq:deviceGroups="[mobile/groups/responsive]"
+            cq:lastModified="{Date}2023-01-05T12:29:52.827+05:30"
+            cq:lastModifiedBy="admin"
+            cq:template="/conf/core-components-it/settings/wcm/templates/af-blank-v2"
+            jcr:language="en"
+            jcr:primaryType="cq:PageContent"
+            jcr:title="Adaptive Form V2 (IT)"
+            sling:resourceType="forms-components-examples/components/page">
         <guideContainer
-            fd:version="2.0"
-            fieldType="form"
-            jcr:primaryType="nt:unstructured"
-            sling:resourceType="forms-components-examples/components/form/container">
-            <textinput
-                fieldType="text-input"
+                fd:version="2.0"
                 jcr:primaryType="nt:unstructured"
-                name="textinput1"
-                jcr:title="Text Input 1"
-                sling:resourceType="forms-components-examples/components/form/textinput"/>
-            <textinput2
+                sling:resourceType="forms-components-examples/components/form/container"
+                dorType="none"
+                fieldType="form">
+            <textinput
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Text Input 1"
+                    sling:resourceType="forms-components-examples/components/form/textinput"
                     fieldType="text-input"
+                    name="textinput1"
+                    validationExpression="$field == 'validate'">
+                <fd:rules
```

Comment:
pls mention the rules (in natural language)in the comments somewhere in the test so that they become easier to debug later.